core/src/modules/plugin.ts

```diff
@@ -350,12 +350,15 @@ export namespace plugin {
    */
   export async function replaceTemplateProfileVariables(
     string: string,
-    profile: Profile
+    profile: Profile,
+    strict: Boolean = true
```

Comment:
I think it's lower-case boolean, and you don't need it! The type is determened automatically from the default value you added:

```suggestion
strict = true
```